app/views/payment/PaymentStudentLicensesView.vue

```diff
@@ -10,6 +10,7 @@
       :license-validity-period-in-days="parseInt(price.metadata.licenseValidityPeriodInDays)"
       :i18n-name="price.metadata.i18nName"
       :min-licenses="getMinLicenses(price)"
+      :key="price.id"
```

Comment:
Just noticed this one may be a remnant from some other work, mentioning in case you wanted this in another branch.